src/neha_Patil/BankSystem.java

```diff
@@ -0,0 +1,94 @@
+package neha_Patil;
+
+public class BankSystem {
+	int CreditAmt;
+	int DebitAmt;
+	String Cname;
+	int Totalbalance = 10000;
```

Comment:
Never make any variable hard coded until and unless there is specific requirement.